lib/game.rb

```diff
@@ -0,0 +1,67 @@
+require "pry"
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5], # Middle row
+    [6,7,8], # Bottom row
+    [0,3,6], # First column
+    [1,4,7], # Second column
+    [2,5,8], # Last column
+    [0,4,8], #\
+    [2,4,6]  #/
+  ]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    self.board.turn_count%2 == 0 ? player_1 : player_2
+  end
+
+  def draw?
+    self.board.full? && !self.won?
+  end
+
+  def over?
+    self.draw? || self.won?
+  end
+
+  def won?
+    WIN_COMBINATIONS.find do |combo|
+      self.board.cells[combo[0]] == self.board.cells[combo[1]] &&
+      self.board.cells[combo[1]] == self.board.cells[combo[2]] && self.board.cells[combo[0]] != " "
+    end
+  end
+
+  def winner
+    self.board.cells[self.won?[0]] if self.won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.